src/GameState.cpp

```diff
@@ -39,7 +39,7 @@ void GameState::init_game(int size, float komi) {
     game_history.clear();
     game_history.emplace_back(std::make_shared<KoState>(*this));
 
-    m_timecontrol.set_boardsize(board.get_boardsize());
+    m_timecontrol.set_boardsize(BOARD_SIZE);
```

Comment:
My bet is that this should be hardcoded in m_timecontrol now

Reply:
Hmm true, I'll do that if gcp validates the removing of m_boardsize.